website/content/en/docs/kubebuilder/references/logging.md

```diff
@@ -0,0 +1,273 @@
+
+// Construct a new logr.logger.
+log = zapcr.New(zapcr.UseDevMode(true), zapcr.WriteTo(os.Stdout), zapcr.Encoder(logfmtEncoder))
+
+// Set the controller logger to log, which will
+// be propagated through the whole operator, generating
+// uniform and structured logs.
+logf.SetLogger(logger)
+```
+
+Ensure that the following additional imports are being used:
+
+```Go
+import(
+	...
+	zaplogfmt "github.com/sykesm/zap-logfmt"
+	zapcr "sigs.k8s.io/controller-runtime/pkg/log/zap"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	...
+)
+```
+**NOTE**: For this example, you will need to add the module `"github.com/sykesm/zap-logfmt"` to your project. Run `go get -u github.com/sykesm/zap-logfmt`.
+
+To test, the following print statement can be added in the main function:
+
+`log.Info("Printing at INFO LEVEL")`
+
+#### Output using custom zap logger
+
+```console
+$ operator-sdk run --local
+INFO[0000] Running the operator locally in namespace default. 
+{"level":"info","ts":1587741740.407766,"logger":"global","msg":"Printing at INFO level"}
+{"level":"info","ts":1587741740.407855,"logger":"scoped","msg":"Printing at INFO level"}
+```
+
+By using `sigs.k8s.io/controller-runtime/pkg/log`, your logger is propagated through `controller-runtime`. Any logs produced by `controller-runtime` code will be through your logger, and therefore have the same formatting and destination.
+
+### Setting flags when running locally
+
+When running locally with `operator-sdk run --local`, you can use the `--operator-flags` flag to pass additional flags to your operator, including the zap flags. For example:
+
+```console
+$ operator-sdk run --local --operator-flags="--zap-log-level=debug --zap-encoder=console"`
+```
+
+### Setting flags when deploying to a cluster
+
+When deploying your operator to a cluster you can set additional flags using an `args` array in your operator's `container` spec. For example:
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: memcached-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: memcached-operator
+  template:
+    metadata:
+      labels:
+        name: memcached-operator
+    spec:
+      serviceAccountName: memcached-operator
+      containers:
+        - name: memcached-operator
+          # Replace this with the built image name
+          image: REPLACE_IMAGE
+          command:
+            - memcached-operator
+          args:
+            - "--zap-log-level=debug"
+            - "--zap-encoder=console"
+          imagePullPolicy: Always
+          env:
+            - name: WATCH_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: OPERATOR_NAME
+              value: "memcached-operator"
+```
```

Comment:
should it cover the new layout or the legacy one? 
If it is for the new one so is missing adapt it. Note that the kube is not == here

Reply:
@camilamacedo86 Are you talking about the example YAML?

Reply:
Yes, this code is valid of SDK layout but it came not from kube layout. Should we not do it based on the new layout? Did you check if is really possible follow up all steps here in the new layout?

Reply:
I think what @camilamacedo86 means is that we need to use the manifests scaffolded by the new CLI in our examples, i.e this one:
https://github.com/operator-framework/operator-sdk-samples/blob/master/go/kubebuilder/memcached-operator/config/manager/manager.yaml

Reply:
Yep, tks @hasbro17 I updated the first comment to see if is more clear as well.

Reply:
@bharathi-tenneti if the above comments have not been addressed, please address them.

Reply:
Above comments have been addressed.

Reply:
@bharathi-tenneti Just a heads up: you can click the `Resolve conversation` button to indicate that it's been addressed.